src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/GpNameAndAddress.java

```diff
@@ -0,0 +1,65 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import org.springframework.util.StringUtils;
```

Comment:
The `isEmpty` method in StringUtils is deprecated and Apache has a `isBlank` method we could use. 
 
```suggestion
import org.apache.commons.lang3.StringUtils;
```

I'd understand if we don't want to change this as this class may be gone soon

Reply:
Done